voila/request_info_handler.py

```diff
@@ -26,28 +26,28 @@ def open(self, kernel_id: str) -> None:
             kernel_id (str): Kernel id used by the notebook when it opens
             the websocket connection.
         """
-        QueryStringSocketHandler._waiters[kernel_id] = self
+        RequestInfoSocketHandler._waiters[kernel_id] = self
         if kernel_id in self._cache:
             self.write_message(self._cache[kernel_id])
 
     def on_close(self) -> None:
-        for k_id, waiter in QueryStringSocketHandler._waiters.items():
+        for k_id, waiter in RequestInfoSocketHandler._waiters.items():
             if waiter == self:
                 break
-        del QueryStringSocketHandler._waiters[k_id]
+        del RequestInfoSocketHandler._waiters[k_id]
 
     @classmethod
-    def send_updates(cls: 'QueryStringSocketHandler', msg: Dict) -> None:
-        """Class method used to dispath the query string to the waiting
-        notebook. This method is called in `VoilaHandler` when the query
-        string becomes available.
+    def send_updates(cls: 'RequestInfoSocketHandler', msg: Dict) -> None:
+        """Class method used to dispath the request info to the waiting
+        notebook. This method is called in `VoilaHandler` when the request
+        info becomes available.
         If this method is called before the opening of websocket connection,
         `msg` is stored in `_cache0` and the message will be dispatched when
```

Comment:
```suggestion
        `msg` is stored in `_cache` and the message will be dispatched when
```

Reply:
This is not a line I've changed or part of this PR, but it's a small enough change to put in. Just checking if you're sure this should be changed in this PR.

Reply:
Thanks for fixing it, this is the repo I left from previous PR 😄